packages/cloudcasa/generated-changes/patch/Chart.yaml.patch

```diff
@@ -10,10 +9,12 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/cloudcasa/charts-original/Chart.yaml p
  keywords:
  - backup
  - Catalogic
-@@ -13,3 +13,8 @@
+@@ -12,4 +12,8 @@
+ - email: info@catalogicsoftware.com
    name: catalogicsoftware
  name: cloudcasa
- version: "1.0"
+-version: "1.0"
++version: "1.0.0"
```

Comment:
@PennyScissors can you confirm from the partner that this is okay? This change was required since the partner is not following proper SemVer versioning, which might / should break on a Helm install with Helm >3.5.2.

Even if it does not break, the Helm community could break it at any time (since the version is expected to be strictly SemVer, regardless of whether Helm errors on it now or now) so it is better to fix now either way.